src/main/java/use_cases/PlayerGameInteractor.java

```diff
@@ -31,27 +34,31 @@ public PlayerGameInteractor(Game game, GameState gameState) {
     @Override
     public PlayerGameResponseModel createResponse(PlayerGameRequestModel pgrm) {
         Player currPlayer = findPlayerFromString(pgrm.getPlayerName());
-        if(currPlayer != null){
+        if (currPlayer != null) {
             Card chosenCard = findCard(pgrm.getCardValue(), pgrm.getCardSuit(), currPlayer);
+
             if (chosenCard != null && pgrm.getPlayCardRequest()) {
-            playCardRequestLogic(chosenCard, currPlayer);
-            } else if (pgrm.getPickUpCardRequest()) {
-            pickUpCardRequestLogic(currPlayer);
-            } else if (pgrm.getSkipTurnRequest()) {
+                playCardRequestLogic(chosenCard, currPlayer);
+            } 
+            else if (pgrm.getPickUpCardRequest()) {
+                pickUpCardRequestLogic(currPlayer);
+            } 
+            else {
                 // Requested to skip turn, check if the Player can play a Card first
                 skipTurnLogic(currPlayer);
             }
             // Now we assume that the Game has been successfully updated, but first check that the Player whose
             // turn it IS NOT a ComputerPlayer, as a User should not play as a ComputerPlayer
-            Player newCurrPlayer = currentGame.getCurrentTurn();
-            while ((newCurrPlayer instanceof ComputerPlayer) & !(currentGame.hasWinner())) {
-            computerPlayerLogic((ComputerPlayer) newCurrPlayer);
-            newCurrPlayer = currentGame.getCurrentTurn();
+            Player newCurrPlayer = access.getCurrentTurn();
+            while ((newCurrPlayer instanceof ComputerPlayer) & !(access.hasWinner())) {
+                computerPlayerLogic((ComputerPlayer) newCurrPlayer);
+                newCurrPlayer = access.getCurrentTurn();
             }
             // Return the current state of the game with a response model.
             return new PlayerGameResponseModel(gameState);
-    } else {
-            return new PlayerGameResponseModel(gameState);
+        } 
+        else {
+            return null;
```

Comment:
In this case, the current player is null. This would happen when a game has been created and game controller has called startGame(). Thus I think this would give a nullpointerexception when run so we should change it to return the a new response containing the gamestate. Unless there is a way do that, I don't know if returning null is the best idea here

Reply:
i believe this has been fixed in prior PRs on an alternate branch. the changes i made here were simply formatting related, it is the same functionally